### PR TITLE
[IMPROVEMENT] Changing search field width should be the max width

### LINKED ISSUE #2045: [IMPROVEMENT] Search field width should be the max width
## Describe the bug

Right now search field is showing with other icons. Whci reduces content space. Search field width should be max.

device-2019-02-26-202946

### PR DESCRIPTION
**Pull request by AuraOfDivinity**
@RocketChat/android

Closes #2045

#### Screenshots or GIF for the change:
lmao

### CODE REVIEW

File: app/src/main/java/chat/rocket/android/chatroom/ui/Menu.kt [hunk 1]
```diff
@@ -111,4 +119,16 @@ private fun ChatRoomFragment.setupDetailsMenuItem(menu: Menu) {
             R.string.title_channel_details
     ).setIcon(R.drawable.ic_info_outline_white_24dp)
             .setShowAsAction(MenuItem.SHOW_AS_ACTION_IF_ROOM)
-}
\ No newline at end of file
+}
+
+private fun removeFavoriteMenuItem(menu: Menu){
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/chatroom/ui/Menu.kt]: Wrong formatting.

File: app/src/main/java/chat/rocket/android/chatroom/ui/Menu.kt [hunk 2]
```diff
@@ -111,4 +119,16 @@ private fun ChatRoomFragment.setupDetailsMenuItem(menu: Menu) {
...
+    menu.removeItem(MENU_ACTION_FAVORITE_UNFAVOURITE_CHAT)
+}
+
+private fun removeDetailMenuItem(menu: Menu){
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/chatroom/ui/Menu.kt]: Wrong formatting.

File: app/src/main/java/chat/rocket/android/chatroom/ui/Menu.kt [hunk 3]
```diff
@@ -111,4 +119,16 @@ private fun ChatRoomFragment.setupDetailsMenuItem(menu: Menu) {
...
+    menu.removeItem(MENU_ACTION_SHOW_DETAILS)
+}
+
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/chatroom/ui/Menu.kt]: Unnecessary blank line.

File: app/src/main/java/chat/rocket/android/chatroom/ui/Menu.kt [hunk 4]
```diff
@@ -111,4 +119,16 @@ private fun ChatRoomFragment.setupDetailsMenuItem(menu: Menu) {
...
+
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/chatroom/ui/Menu.kt]: Unnecessary blank line.

File: app/src/main/java/chat/rocket/android/chatroom/ui/Menu.kt [hunk 5]
```diff
@@ -111,4 +119,16 @@ private fun ChatRoomFragment.setupDetailsMenuItem(menu: Menu) {
...
+
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/chatroom/ui/Menu.kt]: Unnecessary blank line.

File: app/src/main/java/chat/rocket/android/chatroom/ui/Menu.kt [hunk 6]
```diff
@@ -111,4 +119,16 @@ private fun ChatRoomFragment.setupDetailsMenuItem(menu: Menu) {
...
+
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/chatroom/ui/Menu.kt]: Unnecessary blank line.

**Review by philipbrito — CHANGES_REQUESTED** [review]

**Review by philipbrito — CHANGES_REQUESTED** [review]
Could you fix the code formatting issues here?

File: app/src/main/java/chat/rocket/android/chatroom/ui/Menu.kt [hunk 7]
```diff
@@ -18,40 +18,44 @@ internal fun ChatRoomFragment.setupMenu(menu: Menu) {
 internal fun ChatRoomFragment.setOnMenuItemClickListener(item: MenuItem) {
     when (item.itemId) {
         MENU_ACTION_FAVORITE_UNFAVOURITE_CHAT -> presenter.toggleFavoriteChatRoom(
-            chatRoomId,
-            isFavorite
+                chatRoomId,
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/chatroom/ui/Menu.kt]: Wrong indentation.

File: app/src/main/java/chat/rocket/android/chatroom/ui/Menu.kt [hunk 8]
```diff
@@ -18,40 +18,44 @@ internal fun ChatRoomFragment.setupMenu(menu: Menu) {
...
+                isFavorite
         )
         MENU_ACTION_SHOW_DETAILS -> presenter.toChatDetails(
-            chatRoomId,
-            chatRoomType,
-            isSubscribed,
-            disableMenu
+                chatRoomId,
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/chatroom/ui/Menu.kt]: Wrong indentation.

File: app/src/main/java/chat/rocket/android/chatroom/ui/Menu.kt [hunk 9]
```diff
@@ -18,40 +18,44 @@ internal fun ChatRoomFragment.setupMenu(menu: Menu) {
...
     }
 }
 
 private fun ChatRoomFragment.setupSearchMessageMenuItem(menu: Menu, context: Context) {
     val searchItem = menu.add(
-        Menu.NONE,
-        Menu.NONE,
-        Menu.NONE,
-        R.string.title_search_message
+            Menu.NONE,
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/chatroom/ui/Menu.kt]: Wrong indentation.

File: app/src/main/java/chat/rocket/android/chatroom/ui/Menu.kt [hunk 10]
```diff
@@ -86,20 +90,20 @@ private fun ChatRoomFragment.setupSearchViewTextListener(searchView: SearchView)
 private fun ChatRoomFragment.setupFavoriteMenuItem(menu: Menu) {
     if (isFavorite) {
         menu.add(
-            Menu.NONE,
-            MENU_ACTION_FAVORITE_UNFAVOURITE_CHAT,
-            Menu.NONE,
-            R.string.title_unfavorite_chat
+                Menu.NONE,
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/chatroom/ui/Menu.kt]: Wrong indentation.

File: app/src/main/java/chat/rocket/android/chatroom/ui/Menu.kt [hunk 11]
```diff
@@ -86,20 +90,20 @@ private fun ChatRoomFragment.setupSearchViewTextListener(searchView: SearchView)
...
         ).setIcon(R.drawable.ic_star_yellow_24dp)
-            .setShowAsAction(MenuItem.SHOW_AS_ACTION_IF_ROOM)
+                .setShowAsAction(MenuItem.SHOW_AS_ACTION_IF_ROOM)
     } else {
         menu.add(
-            Menu.NONE,
-            MENU_ACTION_FAVORITE_UNFAVOURITE_CHAT,
-            Menu.NONE,
-            R.string.title_favorite_chat
+                Menu.NONE,
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/chatroom/ui/Menu.kt]: Wrong indentation.

**Review by philipbrito — CHANGES_REQUESTED** [review]

File: app/src/main/java/chat/rocket/android/chatroom/ui/Menu.kt [hunk 12]
```diff
@@ -105,10 +109,18 @@ private fun ChatRoomFragment.setupFavoriteMenuItem(menu: Menu) {
 
 private fun ChatRoomFragment.setupDetailsMenuItem(menu: Menu) {
     menu.add(
-            Menu.NONE,
```
Comment (GOVINDDIXIT) [inline app/src/main/java/chat/rocket/android/chatroom/ui/Menu.kt]: formatting issue

**Review by GOVINDDIXIT — COMMENTED** [review]
Pls make sure that the tests passes

**Review by philipbrito — APPROVED** [review]
Thank you very much for sending this PR @AuraOfDivinity!
💯

**Pull request merged**